features/support/rummager_helpers.rb

```diff
@@ -0,0 +1,25 @@
+require "gds_api/test_helpers/rummager"
+
+module RummagerHelpers
+  include GdsApi::TestHelpers::Rummager
+
+  def stub_rummager
+    stub_any_rummager_post
+  end
+
+  def assert_policy_published_to_rummager(policy)
+    expected_json = JSON.parse({
```

Comment:
(No need for this to block merging, but an explanatory note for why we have to do this might help.)